devel/automake/Portfile

```diff
@@ -15,6 +15,7 @@ revision            1
 categories          devel
 platforms           darwin freebsd
 supported_archs     noarch
+configure.cxx_stdlib
```

Comment:
Not sure you meant to include this in this PR ... guessing it is "collateral changes".

Reply:
Was in the original PR from @jmroot .